app/models/debate.rb

```diff
@@ -164,4 +165,15 @@ def self.debates_orders(user)
     orders << "recommendations" if Setting["feature.user.recommendations_on_debates"] && user&.recommended_debates
     orders
   end
+
+  def description_sanitized
+    real_description_length = ActionView::Base.full_sanitizer.sanitize("#{description}").squish.length
+    if real_description_length < 10
+      errors.add(:description,  I18n.t("errors.messages.too_short.other", count: Debate.description_min_length))
```

Comment:
Maybe we could simplify the code a bit:

```suggestion
      errors.add(:description, :too_short, count: Debate.description_min_length)
```